test-suite/shared/src/test/scala/org/scalajs/testsuite/javalib/net/URITest.scala

```diff
@@ -341,6 +341,10 @@ class URITest {
       assertEquals(new URI("http://example.com/asdf%6a").hashCode,
           new URI("http://example.com/asdf%6A").hashCode)
     }
+    assertEquals(new URI("http://example.com").hashCode(),
+      new URI("http://Example.CoM").hashCode())
+    assertNotEquals(new URI("http://example.com/example-com").hashCode(),
+      new URI("http://Example.CoM/eXAMplE-cOm").hashCode())
```

Comment:
It seems these tests need some maintenance:

- `executingInJVM` should be dropped (we dropped JDK6/7 support).
- Tests for `equals` and `hashCode` really should use the same test cases since the specification is so intimately related.

I'd be great if you could add another commit for this.

Reply:
Dropped `executingInJVM` in https://github.com/scala-js/scala-js/pull/3690/commits/065ef673dd763d427160d3365b1eaf4f76f55adb

Refactoring tests for `equals` and `hashCode` in https://github.com/scala-js/scala-js/pull/3690/commits/b9717e97663d6d88eb9184128be7d96e41cf4d01